Dockerfile

```diff
@@ -0,0 +1,96 @@
+# Base container that is used for both building and running the app
+FROM centos:latest as base
+ENV RUBY_SCL="rh-ruby25"
```

Comment:
The shipped ruby version is quite outdated and has known bugs. There is a bugzilla to upgrade to the latest ruby version: https://bugzilla.redhat.com/show_bug.cgi?id=1688754
Can we use a base image that ships a newer ruby version?

Reply:
while that's possible, I was trying to reduce deviation form the current ruby versions we use (for test matrix etc) regardless, I didn't use a ruby based image as we also need nodejs.

Reply:
I went ahead and used fedora:30 with ruby 2.6 / npm 11 using dnf modules, i left it outside of this PR at https://github.com/ohadlevy/foreman/commit/d0e030854565851c6d280a968550aecc282240b7 because i didn't use it yet (just confirmed the app loads correctly).

if this is the direction you had in mind let me know and I'll continue working on that branch, the bonus points is that it will be compatible with centos 8 once its released..